dashboard/api.py

```diff
@@ -187,96 +182,48 @@ def get_info_for_course(user, course, user_enrollments, user_certificates):
     # remove the picked run_status from the list
     run_statuses.remove(run_status)
 
+    def add_run(next_run, status, certificate=None):
+        """Helper function to add a course run to the status dictionary"""
+        course_data['runs'].append(
+            format_courserun_for_dashboard(
+                next_run,
```

Comment:
also just curious, is there any performance downside to declaring this inside another function? I was thinking instead it could take the `course_data` as a parameter and mutate it.